OA-PASS/pass-ember
# Add validation to embargo

## Issues addressed https://github.com/OA-PASS/pass-ember/issues/242
- Add validation to embargo when the embargo checkbox is checked, Embargo enddate is a required field.
- Move Title, Journal Title, and Authors out of NIM schema and in to Common.
- QA code
- Add notice icon to ERIC repos submissions to better indicate action required from user. 
#### Required
![image](https://user-images.githubusercontent.com/17414914/39818750-9c68f620-536f-11e8-8023-f300ae157637.png)
#### Optional
![image](https://user-images.githubusercontent.com/17414914/39818780-af782b5a-536f-11e8-9f68-432e4e2bb62d.png)

## Known bug
- There are two JScholarships showing up on the repo page.
![image](https://user-images.githubusercontent.com/17414914/39818702-86df94d0-536f-11e8-999e-578467d4f9c7.png)
